src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/Server/SqlDataRecord.cs

```diff
@@ -871,13 +862,20 @@ internal void ThrowIfInvalidOrdinal(int ordinal)
                 throw ADP.IndexOutOfRange(ordinal);
             }
         }
+
         private void EnsureSubclassOverride()
         {
-            if (null == _recordBuffer)
+            if (_recordBuffer == null)
             {
                 throw SQL.SubclassMustOverride();
             }
         }
+
+        /// <include file='../../../../../../../../doc/snippets/Microsoft.Data.SqlClient.Server/SqlDataRecord.xml' path='docs/members[@name="SqlDataRecord"]/System.Data.IDataRecord.GetData/*' />
+        IDataReader System.Data.IDataRecord.GetData(int ordinal)
+        {
+            throw ADP.NotSupported();
+        }
```

Comment:
Just inserting the removed attribute would be useful; Could you include it in Net core too?
```suggestion
        /// <include file='../../../../../../../../doc/snippets/Microsoft.Data.SqlClient.Server/SqlDataRecord.xml' path='docs/members[@name="SqlDataRecord"]/System.Data.IDataRecord.GetData/*' />
        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
        IDataReader System.Data.IDataRecord.GetData(int ordinal)
        {
            throw ADP.NotSupported();
        }
```

Reply:
Done in core and netfx.